### PR TITLE
Fix: [Core] Nested Tabs / Remove Duplicated CSS ID [ED-10533]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 1]
```diff
@@ -157,7 +158,7 @@ export default class NestedTabs extends Base {
 		};
 	}
 
-	onHideTabContent( $activeContent ) {}
```
Comment (hein-obox) [inline modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:160]: I have removed this unused variable.

**Review by ArmanPurtoian — DISMISSED** [review]

File: modules/nested-tabs/assets/js/frontend/utils.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+export function isMenuInDropdownMode( elementSettings ) {
```
Comment (louiswol94) [inline modules/nested-tabs/assets/js/frontend/utils.js:1]: @hein-obox This is duplicated code. You can rather move the mega-menu file somewhere more general and import it in both places.

**Review by louiswol94 — CHANGES_REQUESTED** [review]

File: modules/nested-tabs/assets/js/frontend/utils.js (hunk 2 above)
Comment (hein-obox) [reply modules/nested-tabs/assets/js/frontend/utils.js:1]: @louiswol94 

In my version there was a slight difference between the versions, but I have made them identical now.

Proposal: I can remove the version inside the MM in the pro version of this task.
In that way the base function is available in the Core plugin inside the Nested Tabs widget, and is inherited inside the MM widget in the pro version.

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js [hunk 3]
```diff
@@ -368,6 +374,38 @@ export default class NestedTabs extends Base {
 		}
 	}
 
+	setResponsiveTabTitleId( resizeEvent = false ) {
```
[the remark this answers is not in the document]
Comment (hein-obox) [reply modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:377]: Aviv asked David and me to decide on the implementation. 

We think it is best if we have the ID available on all devices, without and prefixes or suffices.

File: modules/nested-tabs/assets/js/frontend/utils.js (hunk 2 above)
Comment (louiswol94) [reply modules/nested-tabs/assets/js/frontend/utils.js:1]: @hein-obox Sorry I forgot this is in core. But okay cool, thanks.

File: modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (louiswol94) [reply modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:377]: @hein-obox I understand. I am not sure I like this approach though :-)

And also. if we go with it, should you not unbind the event?

[the remark this answers is not in the document]
Comment (hein-obox) [reply modules/nested-tabs/assets/js/frontend/handlers/nested-tabs.js:391]: Unbinding added.

Approach: I feel like no solution will be really nice until we remove the duplicated mobile tab titles.

What is your suggestion?

**Pull request closed without merging**